bin/mk-notes

```diff
@@ -8,7 +8,7 @@ my $copy = 0;
 my $in_ul = 0;
 while ( <STDIN> ) {
     chomp;
-    if (/^\s*(Major changes between|Known issues in).*(\d+\.\d+\.\d+)\D.*\[(in pre-release|\d+\s(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s\d+)\]:?$/) {
+    if (/^\s*(Major changes between|Known issues in).*(\d+\.\d+\.\d+)\D.*\[(in pre-release|under development|\d+\s(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s\d+)\]:?$/) {
```

Comment:
To get the a result that does corresponds to what NEWS actually says, I believe you need to insert the following below this line:

``` perl
        next unless $4;
```

Reply:
That should mean it will stop the process only at *dated* titles.

Reply:
``next unless $4;`` will prevent the 1.1.1 "in pre-release" notes from being generated. I have another solution if you still want the "in pre-release" notes.